packages/client/src/components/app/forms/CodeScanner.svelte

```diff
@@ -20,7 +21,7 @@
   let cameraEnabled
   let cameraStarted = false
   let html5QrCode
-  let cameraSetting = { facingMode: "environment" }
+  let cameraSetting = { facingMode: {preferredCamera} }
```

Comment:
This is now an object and won't work - we just want a string.

This needs to be:
```js
let cameraSetting = { facingMode: preferredCamera }
```